tests/raftstore/pd.rs

```diff
@@ -522,6 +522,24 @@ impl TestPdClient {
         });
     }
 
+    pub fn half_split_region(&self, src_region: metapb::Region) {
+        self.set_rule(box move |region: &metapb::Region, _: &metapb::Peer| {
```

Comment:
Please fix merge conflicts.

Reply:
It's blocked by pingcap/kvproto#229  @disksing